src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -217,6 +243,7 @@ private String createTimerName(final HttpRequest request) {
     private static final ProcessorsV2Factory TELEMETRY_V2_FACTORY = new ProcessorsV2Factory();
     private static final String TELEMETRY_STREAM_V1_PATH = "/telemetry/v1/stream";
     private static final String TELEMETRY_STREAM_V2_PATH = "/telemetry/v2/stream";
+    private static final String V1_COLLECTD_SOURCE_PREFIX = "/source/v1/collectd";
```

Comment:
Rename to COLLECTD_V1_SOURCE_PREFIX

The path seems a little backwards from a REST perspective. You are not POST-ing or PUT-ing to a "source" resource; you as the POST-er or PUT-er is the source. How about "/samples/v1/collectd" or "/metrics/v1/collected" or "/scopedmetrics/v1/collectd" -- something like that?